src/main.rs

```diff
@@ -188,11 +192,60 @@ fn record(
                 std::process::exit(1);
             }
             eprintln!("Interrupted.");
-            // Trigger the end of the loop
+            // Trigger the end of the loop in all the threads
             done_clone.store(true, Ordering::Relaxed);
         }).expect("Error setting Ctrl-C handler");
     }
 
+    for pid in pids {
+        let format = format.clone();
+        let home_path = home_path.clone();
+        let maybe_filename = maybe_filename.clone();
+        let done = done.clone();
+
+        let child_pid = std::thread::spawn(move || {
+            record(
+                format.outputter(),
+                &output_filename(&home_path, maybe_filename).expect("Error with output file"),
+                pid,
+                sample_rate,
+                maybe_duration,
+                done
+            ).expect("Failure in record");
```

Comment:
would be good to have better error handling here -- in general it's good to minimize use of `expect`. Maybe we could send the error back to the main thread and then return it from `parallel_record`?

Reply:
From https://doc.rust-lang.org/1.17.0/std/thread/fn.spawn.html it seems like panicing might be acceptable in this case:
> If the child thread panics, join will return an Err containing the argument given to panic.